scripts/mopt

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+mopt:
+  MetaOpt cli script for asynchronous distributed optimization
+
+"""
+from __future__ import print_function, absolute_import
+import getpass
+import datetime
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
+
+from metaopt.io import parsing
+
+if __name__ == '__main__':
```

Comment:
So:
1. instead of using 'kwarg' `scripts=['scripts/mopt']`, you say that we should use:
```
entry_points={
        'console_scripts': ['mopt = metaopt.scripts.main:main']
    }
```
2. and that that I should move `/scripts` folder inside `/src/metaopt`.
3. `main` as a named function

Questions:
* Should `metaopt.scripts` be enlisted as a package in `/setup.py`?
* What is the difference between using `scripts=[]` and using `entry_points={'console_scripts':[]}`??
* Would that remove any need for using `scripts=[]` at all? In which cases are they still useful?

Reply:
scripts=[] would not be required anymore unless we add others scripts. One of the main advantage for entry_points=[] vs scripts=[] is that we can more easily test and reuse functions in the script. I find scripts=[] useful for things like bash scripts. For python, it's better if we can access functions directly and interpret then as scripts using entry_points.